runtime/heiko/src/lib.rs

```diff
@@ -1351,6 +1379,7 @@ construct_runtime!(
         // Loans
         Loans: pallet_loans::{Pallet, Call, Storage, Event<T>} = 50,
         Prices: pallet_prices::{Pallet, Storage, Call, Event<T>} = 51,
+        Crowdloans: pallet_crowdloans::{Pallet, Call, Storage, Config, Event<T>} = 52,
```

Comment:
I holp to use another Id for CrowdLoan's own series.

Reply:
@yz89 please let me know what ID to use and I'll update 🙂

Reply:
@GopherJ any thoughts on what id should be used here? Thanks 🙏

Reply:
actually I think 52 should be fine